src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -937,6 +937,17 @@ public function register(string $id, ?string $class = null): Definition
         return $this->setDefinition($id, new Definition($class));
     }
 
+    /**
+     * Registers a service definition.
+     *
+     * This method allows for simple registration of service definition
+     * with a fluid interface.
+     */
+    public function registerChild(string $id, string $parent): Definition
```

Comment:
this should probably have a return type of `ChildDefinition`

Reply:
Done